web-ui/src/pages/HomePage.jsx

```diff
@@ -27,6 +28,15 @@ export default function HomePage() {
   const [showMyAnniversary, setShowMyAnniversary] = useState(false);
   const [showMyBirthday, setShowMyBirthday] = useState(false);
 
+  let doubleCelebration;
+
+  useEffect(() => {
+    doubleCelebration =
+      myBirthday && me && myAnniversary && showMyBirthday && showMyAnniversary;
+  }, [
+    myBirthday && me && myAnniversary && showMyBirthday && showMyAnniversary,
```

Comment:
I think this line should be a comma-separated list of variables, not and-ed together.